spec/draft/API_specification/function_and_method_signatures.rst

```diff
@@ -5,7 +5,7 @@ Function and method signatures
 
 Function signatures in this standard adhere to the following:
 
-1. Positional parameters must be `positional-only <https://www.python.org/dev/peps/pep-0570/>`_ parameters.
+1. Positional parameters are `positional-only <https://www.python.org/dev/peps/pep-0570/>`_ parameters.
```

Comment:
I think it is good to clarify that there are exceptions to these rules, but I don't think we want to change the normative nature of the current guidance. Meaning, I suggest leaving as `must be` and then just linking to the footnote (or inlining) that there are exceptions due to, e.g., backward compatibility concerns. For future API design, IMO, it is better to keep guidance as normative, rather than descriptive ("are").

Reply:
Perhaps the change here should be s/must be/should be/ in order to allow some normative wiggle room for exceptions.

Reply:
"should be" works I think. "must" is too strong, given both the exceptions to the general rule and that it's not that critical for implementers to do this - they're typically large structural backwards compat breaks, and not necessary to allow users to do the idiomatic thing.